Description: Remove shebang from non-executable files
 Note that upstream considers cli.js as executable.
Bug-npm:https://github.com/isaacs/npm/issues/1521
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2011-10-10
--- a/cli.js
+++ b/cli.js
@@ -1,2 +1 @@
-#!/usr/bin/env node
 require("./bin/npm.js")
--- a/bin/npm-get-uid-gid.js
+++ b/bin/npm-get-uid-gid.js
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
 var argv = process.argv.slice(2)
   , user = argv[0] || process.getuid()
   , group = argv[1] || process.getgid()
--- a/bin/read-package-json.js
+++ b/bin/read-package-json.js
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
 var argv = process.argv
 if (argv.length < 3) {
   console.error("Usage: read-package.json <file> [<fields> ...]")
